### PR TITLE
update alerts dashboard

### PR DESCRIPTION
**Pull request by jcolladokuri**
**What this PR does / why we need it**: Updates alerts dashboard to the latest version and cleans up the dashboard JSON

**Which issue(s) this PR fixes**: NA

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/v1Alerts.json [hunk 1]
```diff
@@ -1042,9 +1005,7 @@
     "hidden": false,
     "refresh_intervals": ["30m", "1h", "12h", "24h", "3d", "7d", "30d"]
   },
-  "timezone": "",
-  "title": "Azure Alert Consumption",
+  "title": "Azure / Alert Consumption",
```
Comment (andresmgot) [inline public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/v1Alerts.json:1008]: I'd keep the naming consistent, so you either leave it as it was or do Azure - Alert Consumption, to match the rest:

Screenshot from 2022-06-14 11-02-35

File: public/app/plugins/datasource/grafana-azure-monitor-datasource/plugin.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     { "type": "dashboard", "name": "Azure SQL Database Insights", "path": "dashboards/sqldb.json" },
     { "type": "dashboard", "name": "Azure Data Explorer Insights", "path": "dashboards/adx.json" },
     { "type": "dashboard", "name": "Azure Resource Overview", "path": "dashboards/arg.json" },
-    { "type": "dashboard", "name": "Azure Alert Consumption", "path": "dashboards/v1Alerts.json" },
+    { "type": "dashboard", "name": "Azure / Alert Consumption", "path": "dashboards/v1Alerts.json" },
```
Comment (andresmgot) [inline public/app/plugins/datasource/grafana-azure-monitor-datasource/plugin.json:13]: same about the name here

**Review by andresmgot — COMMENTED** [review]
Thanks @jcolladokuri, I have some suggestions.

File: public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/v1Alerts.json [hunk 3]
```diff
@@ -93,7 +71,7 @@
                   {
                     "targetBlank": false,
                     "title": "",
-                    "url": "d/dyzn5SK7z/alert-consumption?${sub:queryparam}﻿﻿﻿﻿﻿&﻿﻿﻿﻿﻿${rg:queryparam}﻿﻿﻿﻿﻿&﻿﻿﻿﻿﻿${__url_time_range}﻿﻿﻿﻿﻿&var-mc=Fired&var-mc=Resolved&var-as=New&var-as=Acknowledged&var-as=Closed&var-sev=Sev0&var-sev=Sev1&var-sev=Sev2&var-sev=Sev3&var-sev=Sev4&${__url_time_range}"
+                    "url": "d/dyzn5SK7z/alert-consumption?${sub:queryparam}     &     ${rg:queryparam}     &     ${__url_time_range}     &var-mc=Fired&var-mc=Resolved&var-as=New&var-as=Acknowledged&var-as=Closed&var-sev=Sev0&var-sev=Sev1&var-sev=Sev2&var-sev=Sev3&var-sev=Sev4&${__url_time_range}"
```
[the remark this answers is not in the document]
Comment (jcolladokuri) [reply public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/v1Alerts.json]: thank you for pointing this out, not sure why the spaces got added

File: public/app/plugins/datasource/grafana-azure-monitor-datasource/plugin.json (hunk 2 above)
Comment (jcolladokuri) [reply public/app/plugins/datasource/grafana-azure-monitor-datasource/plugin.json:13]: done

File: public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/v1Alerts.json (hunk 1 above)
Comment (jcolladokuri) [reply public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/v1Alerts.json:1008]: Fixed this

File: public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/v1Alerts.json [hunk 4]
```diff
@@ -93,7 +71,7 @@
                   {
                     "targetBlank": false,
                     "title": "",
-                    "url": "d/dyzn5SK7z/alert-consumption?${sub:queryparam}﻿﻿﻿﻿﻿&﻿﻿﻿﻿﻿${rg:queryparam}﻿﻿﻿﻿﻿&﻿﻿﻿﻿﻿${__url_time_range}﻿﻿﻿﻿﻿&var-mc=Fired&var-mc=Resolved&var-as=New&var-as=Acknowledged&var-as=Closed&var-sev=Sev0&var-sev=Sev1&var-sev=Sev2&var-sev=Sev3&var-sev=Sev4&${__url_time_range}"
+                    "url": "d/dyzn5SK7z/alert-consumption?${sub:queryparam}&${rg:queryparam}&${__url_time_range}&var-mc=Fired&var-mc=Resolved&var-as=New&var-as=Acknowledged&var-as=Closed&var-sev=Sev0&var-sev=Sev1&var-sev=Sev2&var-sev=Sev3&var-sev=Sev4&${__url_time_range}"
```
Comment (andresmgot) [inline public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/v1Alerts.json:74]: these URLs are still wrong, as far as I can see, you need to replace in all of them:

```suggestion
                    "url": "d/dyzn5SK7z/azure-alert-consumption?${sub:queryparam}&${rg:queryparam}&${__url_time_range}&var-mc=Fired&var-mc=Resolved&var-as=New&var-as=Acknowledged&var-as=Closed&var-sev=Sev0&var-sev=Sev1&var-sev=Sev2&var-sev=Sev3&var-sev=Sev4&${__url_time_range}"
```

Please, make sure the code works for you

**Review by andresmgot — COMMENTED** [review]
Thanks! one comment more left

File: public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/v1Alerts.json (hunk 4 above)
Comment (jcolladokuri) [reply public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/v1Alerts.json]: Thanks for the catch, I also had to add the data source variable in the URL!

**Review by andresmgot — APPROVED** [review]
LGTM! Thanks

**Pull request merged**